scripts/Update-QDKVersion.ps1

```diff
@@ -41,48 +41,38 @@ $csFiles | ForEach-Object {
     } | Set-Content -Encoding UTF8 $_.Path
 }
 
+function findAndReplace() {
```

Comment:
I'd make the name a bit more descriptive, since it's not _quite_ find-and-replace you see in IDEs normally
```suggestion
function FindPatternAndReplaceQDKVersion() {
```